chromium/browser_type.go

```diff
@@ -304,7 +304,15 @@ func parseArgs(flags map[string]any) ([]string, error) {
 	for name, value := range flags {
 		switch value := value.(type) {
 		case string:
-			args = append(args, fmt.Sprintf("--%s=%s", name, value))
+			var arg string
+			if strings.TrimSpace(value) != "" {
+				arg = fmt.Sprintf("--%s=%s", name, value)
+			} else {
+				// If the value is empty, we don't include it in the args list.
+				// Otherwise, it will produce "--name=" which is invalid.
+				arg = fmt.Sprintf("--%s", name)
+			}
```

Comment:
The logic is sound and I usually wouldn't comment on this but on initial inspection it felt like there was too much going on here. Could it be made simpler with something like:

```go
arg := fmt.Sprintf("--%s", name)
if strings.TrimSpace(value) != "" {
	arg += fmt.Sprintf("=%s", name, value)
}
```

Reply:
`Sprintf` is inefficient. What about? [ec48706](https://github.com/grafana/xk6-browser/pull/1386/commits/ec48706321e811b599d873880184a8c76f079580)